src/FrontendObligationChecker/Services/PageService/PageService.cs

```diff
@@ -162,14 +162,19 @@ private async Task SetAssociationTypeAsync(Page page)
         var sessionPage = sessionJourney?.Pages.SingleOrDefault(x => x.Path == PagePath.TypeOfOrganisation);
         if (sessionPage != null)
         {
-            var answer = sessionPage.Questions.FirstOrDefault(q => q.Key == QuestionKey.TypeOfOrganisation).Answer;
-            page.AssociationType = answer switch
+            var question = sessionPage.Questions.Find(q => q.Key == QuestionKey.TypeOfOrganisation);
```

Comment:
Please add unit test as I think this is change in behaviour

Reply:
updated